modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java

```diff
@@ -116,6 +118,14 @@ public SELF withInitScript(String initScriptPath) {
         return self();
     }
 
+    /**
+     * Initialize Cassandra client with JMX reporting disabled
+     */
+    public SELF withoutJmxReporting() {
```

Comment:
I'm in favour of this change, but please could we alter the semantics? `withJmxReporting(boolean enableJmxReporting)` would be more in line with how we do things normally. 

I think also we could probably afford to have `enableJmxReporting` default to `false` as well; it's _probably_ not going to be a breaking change for anybody, and is generally a performance improvement that many people could benefit from.